recipe_scrapers/joshuaweissman.py

```diff
@@ -0,0 +1,70 @@
+# mypy: allow-untyped-defs
+
+# from pprint import pprint
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import normalize_string
+
+
+class JoshuaWeissman(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "joshuaweissman.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        raise NotImplementedError()
+
+    def yields(self):
+        raise NotImplementedError()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients = [
+            li.get_text() for ul in self.soup.findAll("ul") for li in ul.findAll("li")
+        ]
+
+        ingredients = [
+            normalize_string(ingredient) for ingredient in ingredients if ingredient
+        ]
+
+        return ingredients
+
+    def ingredient_groups(self):
+        # This is the css for it but web site has weird structuring so it doesn't work properly
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            'span[style="font-weight: 700;"]',
```

Comment:
Ah: the problem here is that BeautifulSoup is going to use this as an exact-match on the string contents of the `span` element's `style` attribute.

This could work (although the test data will also need to be updated):

```suggestion
        # This is the css for it but web site has weird structuring so it doesn't work properly
        return group_ingredients(
            self.ingredients(),
            self.soup,
            'span[style="font-weight:700"]',
```